ONSdigital/census-rm-toolbox
# Making Dockerfile Use Non-root User

# Motivation and Context
Changing the Dockerfile to use non-root user rather than root which makes it more secure (someone could gain root access to the Docker host by hacking the application running inside the container)

# What has changed
Created new user in Dockerfile and assigned permission levels. Also optimised the order in which the image is built. Changed the YAML file to accommodate for non-root user and also changed the location for the user for dosql-i-know-what-i-am-doing based on new Docker image. Also removed manifest files from repo as they are duplicated in census-rm-kubernetes

# How to test? 
Clone and build the new Docker image and apply Kubernetes changes and check that everything in GCP works as it should do

# Links
https://trello.com/c/Pd48zUsd
